src/dev/cli_dev_mode/cli_dev_mode.ts

```diff
@@ -164,6 +165,30 @@ export class CliDevMode {
     }
 
     this.subscription = new Rx.Subscription();
+    this.startTime = Date.now();
+
+    this.subscription.add(
+      firstAllTrue(this.devServer.isReady$(), this.optimizer.isReady$())
```

Comment:
We should probably not rely on the optimizer to be "ready", but allow the optimizer entering an "issue" state to count too as this is the time until the dev is no longer waiting. Thoughts @tylersmalley?

Reply:
I think you're right - if there was a failure the time would include the resolution to fix the issue so long as the server wasn't restarted during that time which we don't want. Waiting for either "issue" or "ready" seems sufficient so long as we set the `meta.success` as you did with bootstrap.